lighthouse-core/report/html/templates.html

```diff
@@ -164,11 +164,13 @@
       width: 100%;
       will-change: transform;
     }
+    .lh-narrow .lh-header-bg {
+      height: var(--report-header-height-narrow);
+    }
     .lh-lighthouse {
       position: absolute;
-      top: var(--report-header-height);
-      right: 50%;
-      transform: translate3d(var(--report-width-half), -100%, 0);
```

Comment:
seems like these all changed because `lh-lighthouse` moved inside the header. what does that get us and do we know if there's a reason it was outside it to begin with? :)

Reply:
Should have included a screenshot of the current UI:
![image](https://user-images.githubusercontent.com/6392995/50356037-6e98e500-0505-11e9-8c52-0abe6b564e6e.png)
Since the lighthouse isn't tied to the bottom right of the header container it doesn't scale correctly on small screens and clips off the edge like so!